packages/gigamon/manifest.yml

```
@@ -0,0 +1,63 @@
+format_version: 3.1.3
+name: gigamon
+title: "Gigamon"
+version: 0.1.0
+description: "This Gigamon package integrates with Filebeat to collect and ingest data from Gigamon devices"
+type: integration
+categories:
+  - custom
+conditions:
+  kibana:
+    version: "^8.12.2"
+  elastic:
+    subscription: "basic"
+screenshots:
+  - src: /img/App-Insight-dashboard.png
+    title: Gigamon App Insight Dashboard Screenshot
+    size: 600x600
+    type: image/png
+  - src: /img/Identifier-Analysis-dashboard.png
+    title: Gigamon Identifier Analysis Dashboard Screenshot
+    size: 600x600
+    type: image/png
+  - src: /img/M21-31-dashboard.png
+    title: Gigamon M21-31 Dashboard Screenshot
+    size: 600x600
+    type: image/png
+  - src: /img/PCI-Compliance-dashboard.png
+    title: Gigamon PCI Compliance Dashboard Screenshot
+    size: 600x600
+    type: image/png  
+  - src: /img/Rogue-Activity-dashboard.png
+    title: Gigamon Rogue Activity Dashboard Screenshot
+    size: 600x600
+    type: image/png
+  - src: /img/Security-posture-dashboard.png
+    title: Gigamon Security Posture Dashboard Screenshot
+    size: 600x600
+    type: image/png  
+  - src: /img/Troubleshooting-dashboard.png
+    title: Gigamon Troubleshooting Dashboard Screenshot
+    size: 600x600
+    type: image/png  
+icons:
+  - src: /img/gigamon-logo.svg
+    title: Gigamon logo
+    size: 32x32
+    type: image/svg+xml
+policy_templates:
+  - name: gigamon
+    title: Gigamon AMI json
+    description: Collect json data from Gigamon AMI
+    inputs:
+      - type: http_endpoint
+        title: Collect json data via HTTP Endpoint
+        description: Collect json data via HTTP Endpoint
+        vars:
+          - name: listen_address
+            type: text
+            title: Listen Address
+            description: The bind address to listen for http endpoint connections. Set to '0.0.0.0' to bind to all available interfaces.
+owner:
+  github: elastic/security-service-integrations
+  type: elastic
```

```suggestion
  type: partner
```

Since it is developed by Gigamon(partner) team and not directly by Elastic.